src/Sulu/Bundle/ContentBundle/Command/WebspaceCopyCommand.php

```diff
@@ -535,6 +543,40 @@ protected function updateSmartContentStructure(
         }
     }
 
+    /**
+     * Updates references in structure for content type `teaser_selection`.
+     *
+     * @param array $structureArray
+     * @param PropertyMetadata $property
+     * @param string $localeSource
+     * @param string $localeDestination
+     */
+    protected function updateTeaserSelection(
+        array &$structureArray,
+        PropertyMetadata $property,
+        $localeSource,
+        $localeDestination
+    ) {
+        if (!array_key_exists($property->getName(),
+                $structureArray) || !$structureArray[$property->getName()]['items']) {
+            return;
+        }
+
+        foreach ($structureArray[$property->getName()]['items'] as $key => $teaserItem) {
+            $targetDocumentDestination = $this->getTargetDocumentDestination(
```

Comment:
is here not a check missing todo this only for `type` page?

Reply:
Not sure where the type is saved but it shoud something like:
foreach (...) {
if ('page' !== $teaserItem['type']) {
     continue;
}

Reply:
I couldn't yet find it, but why would you want to limit which type of teasers should be processed? 

The `getTargetDocumentDestination()` method should be able to resolve any (even custom) Teaser types based on the uuid. 
No?

Reply:
It should skip other types like `article` or any other custom entity.

My example is false the type for pages is `content` so it should be something like:

```
foreach (...) {
     if ('content' !== $teaserItem['type']) {
         continue;
     }

     // ...
}
```

Reply:
@alexander-schranz it's now breaking from the loop in case it's not 'content'.

Reply:
looks correct for me. should work for pages (content) still correctly or?

Reply:
yes, it does work. already tested it.